tests/projects/local-kit/packages/pkg-a/pkg-a.spec

```
@@ -0,0 +1,28 @@
+%global _cross_first_party 1
+#%undefine _debugsource_packages
```

You have to escape macros in comments or else they still have an effect:
```suggestion
#%%undefine _debugsource_packages
```

But you may not want that here since there's no actual build happening, so I would just remove the `#`.